README.md

```diff
@@ -1,51 +1,38 @@
-## [Landwind - Tailwind CSS Landing Page](https://demo.themesberg.com/landwind/)
+# Techbrain Landing Page
```

Comment:
Add `https://namespace-team.github.io/techbrain-website` link to the title as we have deployed it to this page

The link is mentioned in https://github.com/namespace-team/techbrain-website/deployments